hack/logging-dump.sh

```diff
@@ -149,6 +149,15 @@ get_pod_logs() {
     echo ------ container: $container
     oc -n $ns logs $pod -c $container | nice xz > $logs_folder/$pod-$container.log.xz || oc logs $pod | nice xz > $logs_folder/$pod.log.xz || echo ---- Unable to get logs from pod $pod and container $container
   done
+
+  if [ "$collector_folder" == "$2" ]	
+  then	
+    collector=fluentd
+    if [[ "${pod}" =~ .*rsyslog.* ]] ; then
+      collector=rsyslog
+    fi
+    oc exec $1 -c $collector logs | nice xz >> $logs_folder/$pod.log.xz
```

Comment:
for rsyslog, add the `--all` flag.  Not sure if you can do that for fluentd `logs` command, if it will be ignored

Reply:
It looks fluentd logs ignore any options other than -f, -h and --help.  So, it'd be safe to add --all here.

Reply:
@richm @nhosoi where does `-all` get added?

Reply:
`oc exec $1 -c $collector logs --all | ...`

Reply:
updated.